app/views/ChooseProvider.js

```diff
@@ -325,7 +325,7 @@ class ChooseProviderScreen extends Component {
               name='AuthoritiesMenu'
               renderer={SlideInMenu}
               style={{ padding: 20 }}>
-              <MenuTrigger>
+              <MenuTrigger disabled>
```

Comment:
Was this change intentional?

Reply:
That is a commit for a different PR, I must have pushed it to this branch on accident. I can go ahead and change it.